generate.py

```diff
@@ -57,29 +59,47 @@ def main():
         residual_channels=wavenet_params['residual_channels'],
         dilation_channels=wavenet_params['dilation_channels'],
         quantization_channels=wavenet_params['quantization_channels'],
-        use_biases=wavenet_params['use_biases'])
+        use_biases=wavenet_params['use_biases'],
+        fast_generation=args.fast_generation)
 
     samples = tf.placeholder(tf.int32)
 
     next_sample = net.predict_proba(samples)
-
-    saver = tf.train.Saver()
+    
+    if args.fast_generation:
+        sess.run(tf.initialize_all_variables())
```

Comment:
Yeah, there are extra variables here that need to be initialized, so this line is necessary.